MAUI/CheckBox/Overview.md

```diff
@@ -6,7 +6,7 @@ description: Learn here about overall key features in Essential Studio for .NET
 control: SfChekBox
 ---
 
-## .NET MAUI CheckBox (SfCheckBox) Overview
+# .NET MAUI CheckBox (SfCheckBox) Overview
```

Comment:
Change like this - Overview of .NET MAUI CheckBox (SfCheckBox)

Reply:
changed